.github/workflows/helm-validation.yml

```
@@ -19,7 +19,19 @@ on:
 
 jobs:
   e2e-tests-with-helm:
+    name: with RING v${{ matrix.ring_version }}
```

this is going to replace the full name, don't you want something like:
```suggestion
    name: e2e-tests-with-helm with RING v${{ matrix.ring_version }}
```

---

btw is space ok in names (if it's used as path for artifacts / codecov ?

---

spaces are ok

---

I added the E2E tests part but not helm as thats the workflow name, same with Kustomize